packages/cli-hydrogen/project.json

```diff
@@ -3,5 +3,49 @@
     "sourceRoot": "packages/cli-hydrogen/src",
     "projectType": "library",
     "implicitDependencies": [],
-    "tags": ["scope:feature"]
+    "tags": ["scope:feature"],
+    "targets": {
+      "clean": {
```

Comment:
Given how similar these are between packages, it's a pity we have to restate them so often. Is there no way some base config could be extended?